File: Swift/find-the-difference.swift
class Solution {
    func findTheDifference(_ s: String, _ t: String) -> Character {
        let s = Array<Character>(s)
        var t = Array<Character>(t)

        for char in s {
            if let index = t.index(of: char) {
                t.remove(at: index)
            }
        }

        return t[0]
    }
}

File: Python/find-the-difference.py
class Solution:
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        s = list(s)
        t = list(t)

        for char in s:
            if char in t:
                t.remove(char)

        return t[0]